spec/shared_examples.rb

```diff
@@ -26,12 +26,19 @@
   its(symbol) { is_expected.to eq(df) }
 end
 
+RSpec.shared_examples 'redis importer' do
```

Comment:
Sure, will port the shared_examples to respective `_spec.rb` file. Regarding the `it_behaves_like` part, I preferred `let`s over passing arguments as the former was DRY. Usually, each example data is tested on a set of input options (a context each) and they inherit missing data with `lets`.